package.json

```diff
@@ -56,7 +57,8 @@
     "typescript": "5.3.3",
     "utils": "workspace:*",
     "vite": "^5.0.13",
-    "vitest": "1.2.2"
+    "vitest": "1.2.2",
+    "vue-eslint-parser": "^9.4.2"
```

Comment:
Do `eslint-plugin-vue` and `vue-eslint-parser` need to be int he root `package.json`? Ideally these would be in the `framework-nuxt/package.json` instead

Reply:
OK, good point. I have moved them.

Reply:
On a second note: we would need this also in the apps/dev/nuxt project as well, right?